test/test_pipeline/test_tabular_classification.py

```diff
@@ -333,7 +341,7 @@ def test_set_range_search_space_updates(self, fit_dictionary_tabular):
         except AssertionError as e:
             # As we are setting num_layers to 1 for fully connected
             # head, units_layer does not exist in the configspace
-            assert 'fully_connected:units_layer' in e.args[0], e.args[0]
+            assert 'fully_connected:units_layer' in e.args[0]
```

Comment:
could you remove this change. Printing e.args[0] is important for debugging the test. So In case the error was not with fully_connected but with something else, we wont be able to see what caused the error

Reply:
Sure, I can do that.